devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusPrepare.java

```diff
@@ -62,36 +62,47 @@ public void prepareQuarkus() {
             final Convention convention = getProject().getConvention();
             JavaPluginConvention javaConvention = convention.findPlugin(JavaPluginConvention.class);
             if (javaConvention != null) {
+                System.out.println("quarkus prepare java convention found");
```

Comment:
This should either be removed or logged using getLogger().

Reply:
yes, I will this one

Reply:
and all other from that file ;)